intake_esm/core.py

```diff
@@ -1,13 +1,15 @@
 import os
 import uuid
 
+import s3fs
 import yaml
 from intake.catalog import Catalog
 from intake.catalog.local import LocalCatalogEntry
 
 from . import config as config
 from .bld_collection_utils import FILE_ALIAS_DICT, load_collection_input_file
 from .cesm import CESMCollection
+from .cesm_aws import CESMAWSCollection
```

Comment:
Naively, I would assert that the build catalog method differs for AWS, but except for reading zarr instead of netCDF, nothing else does.